pkg/kubelet/dockertools/docker.go

```diff
@@ -40,8 +40,9 @@ import (
 )
 
 const (
-	PodInfraContainerName = leaky.PodInfraContainerName
-	DockerPrefix          = "docker://"
+	PodInfraContainerName  = leaky.PodInfraContainerName
+	DockerPrefix           = "docker://"
+	PodInfraContainerImage = "gcr.io/google_containers/pause:0.8.0"
```

Comment:
@vmarmol I have checked about this. The flag is stored in kubelet, I think we should move it to the DockManager?